app/controllers/api/v1/accounts/articles_controller.rb

```diff
@@ -68,4 +78,12 @@ def list_params
   def set_current_page
     @current_page = params[:page] || 1
   end
+
+  def update_positions(positions_hash)
```

Comment:
Can you move this method to `Article.rb`?

Reply:
@tejaswinichile done, can you check